charts/ocis/templates/idm/deployment.yaml

```diff
@@ -18,7 +18,8 @@ spec:
       initContainers:
       {{- if and $.Values.services.idm.persistence.enabled $.Values.services.idm.persistence.chownInitContainer }}
         - name: init-chown-data
-          image: busybox
+          image: {{ template "ocis.initContainerImage" $ }}
+          imagePullPolicy: {{ .Values.initContainerImage.pullPolicy }}
           securityContext:
             runAsNonRoot: false
             runAsUser: 0
```

Comment:
I would put all of that into a template as it seems to be the same at all places.